addons/block_code/serialization/block_name_tree_node.gd

```diff
@@ -0,0 +1,13 @@
+extends Resource
+
+const BlockNameTreeNode = preload("res://addons/block_code/serialization/block_name_tree_node.gd")
```

Comment:
I think we agreed on naming this BlockSerialization. This seems to be a replacement for the current file `addons/block_code/serialization/block_serialization.gd`. It would be nice if you can change that file instead of leaving it abandoned and adding a new one.

Reply:
Okay, just renamed the serialization classes and propagated the changes to commits on top!